src/shogun/kernel/GaussianKernel.h

```diff
@@ -36,20 +36,10 @@ class GaussianKernel: public ShiftInvariantKernel
 {
 	friend class params::GaussianWidthAutoInit;
 public:
-	/** default constructor */
 	GaussianKernel();
 
-	/** constructor
-	 *
-	 * @param width width
-	 */
 	GaussianKernel(float64_t width);
 
-	/** constructor
-	 *
-	 * @param size cache size
-	 * @param width width
-	 */
 	GaussianKernel(int32_t size, float64_t width);
```

Comment:
remove this ctor

Reply:
actualy nevermind. Could you revert all those docstring changes (we can do them later)

Reply:
Yes, I'm making a fource push